src/snowcli/cli/nativeapp/manager.py

```diff
@@ -219,12 +219,18 @@ def get_app_pkg_distribution_in_snowflake(self) -> str:
             )
         )
 
-    def is_app_pkg_distribution_same_in_sf(self) -> bool:
+    def is_app_pkg_distribution_same_in_sf(
```

Comment:
Not a fan of having something called is_<foo> have a side effect that logs a warning. How about we call this check or verify instead? Also, now that it accepts a parameter, the "_in_sf" part isn't necessarily true.